File: nas_spec/code_samples/Node/sessions@{id}@issuer-fingerprint/put.js
import { Checkout } from 'checkout-node-sdk';

let cko = new Checkout('your_client_secret_here', {
	client: 'ack_XXXXXXXXXXXX',
	scope: ['sessions:browser'], // array of scopes
	environment: 'sandbox', // or "production"
});

// Or if you use api keys:
// const cko = new Checkout('sk_sbox_XXX', { pk: 'pk_sbox_XXX'}});

try {
	let updated = await cko.sessions.update3DSMethodCompletionIndicator('sid_aurdb2b3yv6eniu7mbrl7nfopm', 'U');
} catch (err) {
	console.log(err.name);
}

File: nas_spec/code_samples/Go/sessions@{id}@issuer-fingerprint/put.go
// For more information please refer to https://github.com/checkout/checkout-sdk-go
import (
	"github.com/checkout/checkout-sdk-go"
	"github.com/checkout/checkout-sdk-go/common"
	"github.com/checkout/checkout-sdk-go/configuration"
	"github.com/checkout/checkout-sdk-go/sessions"
)

// API Keys
api, err := checkout.
			Builder().
			StaticKeys().
			WithSecretKey("secret_key").
			WithEnvironment(configuration.Sandbox()). // or Environment.PRODUCTION
			Build()
if err != nil {
	return nil, err
}

// OAuth
api, err := checkout.
			Builder().
			OAuth().
			WithClientCredentials("client_id", "client_secret").
			WithScopes([]string{configuration.SessionsApp, configuration.SessionsBrowser}).
			WithEnvironment(configuration.Sandbox()). // or Environment.PRODUCTION
			Build()
if err != nil {
	return nil, err
}

request := sessions.ThreeDsMethodCompletionRequest{ThreeDsMethodCompletion: common.Y}

response, err := api.Sessions.Update3dsMethodCompletion("session_id", request, "session_secret") // "session_secret" is optional
if err != nil {
	return nil, err
}

return response, nil
